CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## Stash v1.0.0 Changelog
+
+### v1.0.0
+
+* Removed deprecated DriverList::getDrivers function.
+
+* Removed deprecated invalidation constants in the Item class.
+
+* Removed SQLite Extension support (SQLite3 is still available).
+
+* The `set` function no longer persists data.
+
+* Removed expiration time for `set` function
+
+* Added `expiresAt` and `expiresAfter` functions to the Item class.
+
+* `getExpiration` to return current datetime when no record exists.
```

Comment:
this sound strange, as this is currently not part of the PSR interface, maybe rather return null if none is set.
Otherwise expiry should probably have been on the `pool->save( $item, $ttl )` method